src/actions/CollectionActions.js

```diff
@@ -8,25 +8,34 @@ function createCollectionSuccess(newCollection) {
   return { type: 'CREATE_COLLECTION_SUCCESS', newCollection };
 };
 
-function deleteCollectionSuccess(id) {
-  return { type: 'DELETE_COLLECTION_SUCCESS', id };
+function deleteCollectionSuccess(name) {
+  return { type: 'DELETE_COLLECTION_SUCCESS', name };
```

Comment:
This is the only change I made in terms of Redux logic. I will explain later but App-server do not keep track of the additional collections in a conventional way. So we would need to use `name` instead of `id` for now.